CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ New feature requests on the legacy SDK will not be accepted.
 ## Module and package structure
 
 The code is structured into several generators inside the [generators folder](generators):
-* [2.0](generators/bdk) generating a bot and using the [Java BDK2.0](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java);
+* [2.0](generators/bdk/java) generating a bot and using the [Java BDK2.0](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java);
```

Comment:
We should update the BDK2.0 repo URL as well to https://github.com/finos/symphony-bdk-java